src/redux/book/bookSlice.js

```diff
@@ -0,0 +1,25 @@
+const ADDBOOK = 'book-store/redux/book/bookSlice/ADDBOOK';
+const REMOVEBOOK = 'book-store/redux/book/bookSlice/REMOVEBOOK';
+
```

Comment:
- [ ] [OPTIONAL]
By convention, a slice file of a component should be named in the plural form with the _'Slice'_ keyword attached to it. Therefore, in the case of book slice, we should have `booksSlice.js` instead of  `bookSlice.js`

![image](https://user-images.githubusercontent.com/12745474/229925812-0368960d-ecbb-4430-bd97-b2808e068920.png)